edit/buildozer.go

```diff
@@ -688,10 +690,18 @@ func rewrite(commandsForFile commandsForFile) *rewriteResult {
 			return &rewriteResult{file: name, errs: []error{err}}
 		}
 	} else {
-		// read a file
-		data, fi, err = file.ReadFile(name)
+		origName := name
+		name = strings.TrimSuffix(name, "BUILD")
```

Comment:
I don't think so. By patching here, we let the rest of the code continue to assume that BUILD files are called `BUILD`. We trim off each suffix to go to the next suffix on line 701.

If we want to avoid letting the rest of the code assume that BUILD files are called `BUILD`, that would seem to entail a substantial reworking of the flow of the tool. I tried that approach originally, but it was much more intensive and seemed to necessitate moving `ReadFile` calls earlier in the flow (so we know which suffix to use).